.circleci/config.yml

```diff
@@ -134,35 +134,64 @@ jobs:
             echo "  console.log(decryptData(input));" >> ~/project/decrypt.js
             echo "}" >> ~/project/decrypt.js
 
-      - run:
-          name: Decode secrets and environment variables
-          command: |
-            if [ -n "<< pipeline.parameters.SECRETS >>" ]; then
-              echo "Downloading and decoding secrets..."
+      - when:
+          condition:
+            not:
+              equal: ["upload", "<< pipeline.parameters.SOURCE_TYPE >>"]
+          steps:
+            - run:
+                name: Decode secrets and environment variables
+                command: |
+                  set -e  # Exit immediately if a command exits with a non-zero status
 
-              # Configure AWS CLI
-              aws configure set aws_access_key_id $AWS_ACCESS_KEY_ID
-              aws configure set aws_secret_access_key $AWS_SECRET_ACCESS_KEY
-              aws configure set default.region $AWS_REGION
+                  if [ -n "<< pipeline.parameters.SECRETS >>" ]; then
```

Comment:
i think you can delete SECRETS as a parameter for this job. you can probably also delete this code, this is the old attempt at allowing secrets

https://github.com/earthfast/dashboard-runner/blob/9dbe963a3ff4528b2068b9293841e7c4647651b1/.circleci/config.yml#L108-L135

https://github.com/earthfast/dashboard-runner/blob/9dbe963a3ff4528b2068b9293841e7c4647651b1/.circleci/config.yml#L147-L166